packages/js/data/src/plugins/types.ts

```diff
@@ -39,6 +39,7 @@ export type Plugin = {
 	recommendation_priority?: number;
 	is_visible?: boolean;
 	is_local_partner?: boolean;
+	is_offline?: boolean;
```

Comment:
Shall we simply add a changelog entry for `data` package?

Reply:
Sounds good! Added in [1fb2418](https://github.com/woocommerce/woocommerce/pull/32467/commits/1fb2418e57d553a3f607b904e9ddf35da1c8d85e)